### ISSUE TITLE
No notification with Windows 10

### ISSUE THREAD

**Issue by emresaracoglu** [jolicode/JoliNotif#43]
> Windows 10
> Mozilla & Chrome
> No Internal 500 Error

The code structure looks like the below.

```php
<?php
	ini_set('display_errors', 1);
	ini_set('display_startup_errors', 1);
	error_reporting(E_ALL | E_STRICT);
	require_once '.../vendor/autoload.php';
	
	use Joli\JoliNotif\Notification;
	use Joli\JoliNotif\NotifierFactory;

// Create a Notifier
	$notifier = NotifierFactory::create();

// Create your notification
	$notification =
		(new Notification())
			->setTitle('Notification title')
			->setBody('This is the body of your notification')
	;

// Send it
	$notifier->send($notification);
```

Where is the error?
Thank you.

**Comment by pyrech** [comment]
Hello :)
You mention Firefox and Chrome but JoliNotif is not meant to be run through a web server. To display a desktop notification, the PHP script should be run in CLI application. Is it your case?

**Comment by emresaracoglu** [comment]
Hello:)
Absolutely but no action for desktop notification

**Comment by pyrech** [comment]
Could you run the provided example (`php vendor/jolicode/jolinotif/example/index.php` - not certain of the path) and report the output? It will help to determine if the problem comes from JoliNotif or from Toaster - the underlying library used on Windows 10

**Comment by emresaracoglu** [comment]
I run and I got it as output:

> No supported notifier 

```
[tryover@server5 ~]$ php /home/tryover/vendor/jolicode/jolinotif/example/index.php
No supported notifier
[tryover@server5 ~]$
```
Also do I need an extra PHP module for this?

**Comment by pyrech** [comment]
No it should work out of the box on Windows. 😉 
Thanks for the output. From what I understand it probably means that [this line](https://github.com/jolicode/JoliNotif/blob/master/src/Notifier/ToasterNotifier.php#L45) is not working correctly.
I do not have a Windows 10 right now, so could not test it. If you could try to debug it (dump and paste each part of the return statement and the inner statement in OsHelper class) that would be awesome.

**Comment by emresaracoglu** [comment]
I do not get any error screens with debug, unfortunately, I do not know what to do for this reason. 
If you want, you can try on my PC with TeamViewer.

**Comment by emresaracoglu** [comment]
Any progress for this issue? If it does not exist, I can turn it off.

**Comment by pyrech** [comment]
Did not look at it yet. Thanks for the reminder, I will try to test it this weekend.

**Comment by pyrech** [comment]
Hi! I just tested JoliNotif (version 1.2.0) on a bare Windows 10 system with PHP 7.1 and I can confirm a notification showed up when executing the embeded `example/index.php`. 

By re-reading your comment https://github.com/jolicode/JoliNotif/issues/43#issuecomment-340296315, it looks like you are on a Linux system..? So I don't really understand your issue :confuses:

Closing for now. Do not hesitate to reopen with more information if you still have the issue.

Thanks
